libs/git_wrapper/Git_wrapper.ml

```diff
@@ -193,7 +193,10 @@ let temporary_remote_checkout_path caps url =
   let rand_prefix = Uuidm.v `V4 |> Uuidm.to_string in
   let name = rand_prefix ^ "_" ^ name in
   let tmp_dir = CapTmp.get_temp_dir_name caps#tmp in
-  Fpath.add_seg tmp_dir name
+  let fpath = Fpath.add_seg tmp_dir name in
```

Comment:
Is there not a better way to get a tempdir?

Reply:
this function is unused, so Just gonna delete all this